.github/workflows/che-variables-update.yml

```
@@ -0,0 +1,37 @@
+# This Workflow performs an update of Che variables
+name: Update Che variables in the docs
+on:
+  workflow_dispatch:
+    inputs:
+      branch:
+        description: 'Branch in which to update variables'
+        required: true
+        default: ''
+jobs:
+  build:
+    name: Create Release
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Create Release
+        run: |
+          git config --global user.name "Mykhailo Kuznietsov"
+          git config --global user.email "mkuznets@redhat.com"
+          export GITHUB_TOKEN=${{ secrets.CHE_BOT_GITHUB_TOKEN }}
+
+          BASE_BRANCH=${{ github.event.inputs.branch }}
+          git checkout ${BASE_BRANCH}
+          ./tools/environment_docs_gen.sh
+          
```

Add also the other script:

```suggestion
          ./tools/checluster_docs_gen.sh
```

---

Oh no really only environment_docs_gen.sh! I should have read the PR description more carefully.